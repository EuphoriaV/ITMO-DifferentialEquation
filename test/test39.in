5.12
(((((3.28 * (x * (x * 8.79))) + (x * 4.54)) * ((6.57 * 1.3) + x)) * (4.13 - 5.25)) * (((5.48 + 9.71) * 9.59) - 4.18))
4.23
1.22