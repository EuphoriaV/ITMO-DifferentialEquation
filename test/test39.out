-9.24E-5